tests/vmi_multus_test.go

```diff
@@ -881,6 +869,82 @@ var _ = Describe("[Serial]SRIOV", func() {
 				return libnet.PingFromVMConsole(vmi2, cidrToIP(cidrA))
 			}, 15*time.Second, time.Second).Should(Succeed())
 		})
+
+		Context("VLANed SR-IOV network with a vlaned VMI", func() {
+			var (
+				macVlaned1 string
+			)
+
+			const (
+				cidrVlaned1          = "192.168.0.1/24"
+				sriovVlanNetworkName = "sriov-vlan"
+			)
+
+			generateRandomMacAsString := func() string {
+				mac, err := tests.GenerateRandomMac()
+				Expect(err).ToNot(HaveOccurred())
+				return mac.String()
+			}
+
+			BeforeEach(func() {
+				macVlaned1 = generateRandomMacAsString()
+				createNetworkAttachementDefinition(sriovVlanNetworkName, tests.NamespaceTestDefault, sriovConfVlanCRD)
+				// create an anonymous vmi to be used for pinging
+				createSriovVMIIWithStaticIPByMac(sriovVlanNetworkName, macVlaned1, cidrVlaned1)
+			})
+
+			Context("and a second Vlaned VMI on the same Vlan", func() {
+				var (
+					vlanedVMI2 *v1.VirtualMachineInstance
+					macVlaned2 string
+				)
+
+				const cidrVlaned2 = "192.168.0.2/24"
+
+				BeforeEach(func() {
+					macVlaned2 = generateRandomMacAsString()
+					vlanedVMI2 = createSriovVMIIWithStaticIPByMac(sriovVlanNetworkName, macVlaned2, cidrVlaned2)
+				})
+
+				It("[test_id:3958]should be able to ping between two VMIs with the same VLAN over SRIOV network", func() {
```

Comment:
https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3479/pull-kubevirt-e2e-kind-1.17-sriov/1334049146956419074

Reply:
This test actually should fail. I accidentally changed it to shoudl(succeed())). Thank you for  this:)

about the numbers, I am not sure. We numbers all over this test file. maybe @phoracek  can say if we can leave them.